docs/source/api/apollo-server.md

```diff
@@ -115,7 +115,7 @@ new ApolloServer({
 
 ## registerServer
 
-The `registerServer` method is from `apollo-server-express`. Middleware registration has been greatly simplified with this new method.
+The `registerServer` method providec by all integrations. Middleware registration has been greatly simplified with this new method.
```

Comment:
"provided". also it's a function, not a method.

Also I'm not sure that documenting something as simplifying something with a new method will age well?  Maybe just be more explicit about what it does, like "This function is what lets you connect your ApolloServer to your web framework. Use it instead of the `listen` method when you want to customize your app's web behavior." 
or something